student.rb

```diff
@@ -1,4 +1,7 @@
+require_relative 'high_five'
+
 class Student
+  include HighFive
   attr_reader :age, :phase
   attr_accessor :name
```

Comment:
These all appear in the teacher classes too!  How can you refactor these similarities?